Microsoft.Toolkit.Uwp.Samples.BackgroundTasks/Microsoft.Toolkit.Uwp.Samples.BackgroundTasks.csproj

```diff
@@ -137,6 +140,53 @@
   <PropertyGroup Condition=" '$(VisualStudioVersion)' == '' or '$(VisualStudioVersion)' &lt; '14.0' ">
     <VisualStudioVersion>14.0</VisualStudioVersion>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'OneDrive|AnyCPU'">
```

Comment:
Need to remove OneDrive configuration from here as well.

Reply:
I'm not sure why this remained - I was trying to speed up my build by constraining it to just the OneDrive bits - I deleted the config so it shouldn't have been here - I'll stake it down and sprinkle it with some holy water this time.